include/libesphttpd/httpd.h

```diff
@@ -32,9 +32,13 @@ extern "C" {
 #define HTTPD_MAX_POST_LEN		2048
 #endif
 
-//Max send buffer len
-#ifndef HTTPD_MAX_SENDBUFF_LEN
-#define HTTPD_MAX_SENDBUFF_LEN	2048
+//Send buffer size
+#ifndef HTTPD_SENDBUFF_SIZE
+#define HTTPD_SENDBUFF_SIZE	2048
+#endif
+
+#ifndef HTTPD_SENDBUFF_MAX_FILL
+#define HTTPD_SENDBUFF_MAX_FILL	HTTPD_SENDBUFF_SIZE - 2
```

Comment:
can you add a comment as to why the -2 is here? I'm pretty sure I know why but I'm thinking about the next person looking at these #defines

Reply:
Sure. Added in ec01c41.